src/argparse.ts

```diff
@@ -2271,6 +2271,60 @@ export const CLI_ARGS = {
       '    ]\n' +
       '\n',
       group: 'Peer Services'
+    },
+    get_did_configuration: {
+      type: 'array',
+      items: [
+        {
+          name: 'blockstack_id',
+          type: 'string',
+          realtype: 'blockstack_id',
+          pattern: NAME_PATTERN + '|'+ SUBDOMAIN_PATTERN
+        },
+        {
+          name: 'domain',
+          type: 'string',
+          realtype: 'domain',
+          pattern: NAME_PATTERN + '|'+ SUBDOMAIN_PATTERN
+        },
+        {
+          name: 'owner_key',
+          type: 'string',
+          realtype: 'private_key',
+          pattern: `${PRIVATE_KEY_PATTERN}`
+        },
+      ],
+      minItems: 3,
+      maxItems: 3,
+      help: 'Creates a DID configuration for the given blockstack id and domain to create a link between both.' +
+      'The specification is define by the Decentralized Identity Foundation at https://identity.foundation/specs/did-configuration/\n' +
+      'The DID configuration should be placed in the json file ".well_known/did_configuration"' +
+      '\n'+
+      'Example:\n'+
+      '    $ blockstack-cli get_did_configuration public_profile_for_testing.id.blockstack helloblockstack.com PRIVATE_OWNER_KEY\n' +
```

Comment:
Can you use a raw private key here, or add something like `$ export PRIVATE_OWNER_KEY="6687687251c97f8718bdd9f568e7fefde97549dcbee7fbe89e136a0b7aad448201"` above it?  We try to make sure that a user can copy/paste examples line-by-line and have them "just work".

Reply:
Done